modules/bidmaticBidAdapter.js

```diff
@@ -0,0 +1,101 @@
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { replaceAuctionPrice, isNumber, deepAccess } from '../src/utils.js';
+
+export const END_POINT = 'https://adapter.bidmatic.io/ortb-client';
+const BIDDER_CODE = 'bidmatic';
+const DEFAULT_CURRENCY = 'USD';
+
+export const converter = ortbConverter({
+  context: {
+    netRevenue: true,
+    ttl: 290,
+  },
+  imp(buildImp, bidRequest, context) {
+    const imp = buildImp(bidRequest, context);
+    if (!imp.bidfloor) {
```

Comment:
make sure to use getFloor

Reply:
refactored. I assumed floor module is cross integrated with ortb converter

Reply:
open issue https://github.com/prebid/Prebid.js/issues/11285; would appreciate your thoughts